test_scripts/CloudAppRPCs/005_SetCloudAppProperties_Success.lua

```diff
@@ -0,0 +1,94 @@
+---------------------------------------------------------------------------------------------------
+--  Precondition: 
+--  1) Application with <appID> is registered on SDL.
+--  2) Specific permissions are assigned for <appID> with SetCloudAppProperties
+--
+--  Steps:
+--  1) Application sends a SetCloudAppProperties RPC request
+--  2) Application triggers a PTU
+--  3) Checks policy table to make sure cloud app properties are set correctly
+--
+--  Expected:
+--  1) SDL responds to mobile app with "ResultCode: SUCCESS,
+--        success: true
+--  2) VerifyCloudAppProperties succeeds
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/CloudAppRPCs/commonCloudAppRPCs')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local rpc = {
+  name = "SetCloudAppProperties",
+  params = {
+    properties = {
+      appName = "TestApp",
```

Comment:
This parameter was renamed to `nicknames` and was made an array, should include expectations for this as well.